app/views/audits/audits/_content_preview.html.erb

```diff
@@ -12,4 +12,5 @@
   src: content_item.proxy_url,
   scrolling: "yes",
   frameborder: "no",
+  class: "if-js-hide",
```

Comment:
Out of curiosity, where is this class defined? Is it part of `govuk_admin_template`?

Reply:
[Yup](https://github.com/alphagov/govuk_admin_template/blob/340e2e341b28783d9ee44a60ce4a0cf07af81c3b/CSS.md#hide-and-show-content)